api/test/routes/propositions.routes.test.ts

```diff
@@ -0,0 +1,202 @@
+import { build } from "../helper";
+import bcrypt from "bcrypt";
+import createUser from "../utils/createUser";
+import createMovie from "../utils/createMovies";
+import createSlot from "../utils/createSlot";
+
+describe("Users routes test", () => {
```

Comment:
Change the description

Reply:
Fixed on [#fdbb1c6](https://github.com/Cinevoraces/cinevoraces/pull/41/commits/fdbb1c673f670326c67226ab4aa60a94878c67fb)